File: nowcoder/BC36_健康评估/BC36.go
package main
 
import "fmt"
 
func main() {
    var height,weight,bmi float64
    fmt.Scanf("%f%f",&height,&weight)
    bmi = height / (weight*weight)
    if(bmi>18.5&&bmi<23.9){
        fmt.Printf("Normal")
    }else{
        fmt.Printf("Abnormal")
    }
}

File: nowcoder/BC36_健康评估/BC36.c
#include<stdio.h>
int main(){
    float height,weight,bmi;
    scanf("%f%f",&height,&weight);
    bmi=height/(weight*weight);
    if(bmi>18.5&&bmi<23.9){
        printf("Normal");
    }else{
        printf("Abnormal");
    }
      
    return 0;
}